### PR TITLE
[RFC] Add an explanation to the app_dev.php file

### PR DESCRIPTION
**Pull request by leofeyer**
This PR adds an explanation on how to generate the access key in the `app_dev.php` file.

@aschempp Are you ok with my changes?

### CODE REVIEW

**Comment by aschempp** [comment]
Well in my original version both username and password were separately encoded to sha512. But I guess your version is totally fine too. However, I would prefer to add an extensive description to our new manual (e.g. https://docs.contao.org/books/contao4-migration/) and not a very short version in the document.

**Comment by leofeyer** [comment]
Why not have both the short version and the chapter in the manual?

**Comment by aschempp** [comment]
That would be ok too, but you should not remove the FIXME…

File: web/app_dev.php [hunk 1]
```diff
@@ -12,31 +12,31 @@
 use Symfony\Component\Debug\Debug;
 
 // Access to debug front controllers is only allowed on localhost or with username and password.
-// FIXME: add link to documentation that explains what to do
+// The username and password need to be separated by a colon and then converted to a SHA512 hash.
```
Comment (xchs) [inline web/app_dev.php:15]: What would happen if the password itself contains a colon?

Comment (aschempp) [inline web/app_dev.php:15]: Good point. For the password this could be fixed by using argument `2` for `explode`, but not for the username.

Comment (leofeyer) [inline web/app_dev.php:15]: It does not matter at all. The string could also be `admin:my:pass:word` as long as the hash can be validated in line 34.

**Pull request merged**